src/events/eventToAction.js

```diff
@@ -125,7 +126,11 @@ export default (state, event) => {
         selfEmail: state.accounts[0].email,
         type: opToActionTyping[event.op],
       };
-
+    case 'muted_topics':
+      return {
+        type: EVENT_MUTED_TOPICS,
+        muted_topics: event.muted_topics
```

Comment:
Here you can do either:
```js
...events,
```
And achieve the same + if there are any other props they will be passed on...
or do 
```js
mutedTopics: event.muted_topics
```
since all the naming we custom create will be in js casing.